src/Components/Filters/Search.jsx

```diff
@@ -1,10 +1,16 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import {
-  InputGroup, DropdownButton, Dropdown, FormControl,
-} from 'react-bootstrap';
 
 import Styles from './Search.styles.scss';
+import Dropdown from './../Common/Dropdown';
+
+const SEARCH_CATEGORY = [{
+  key: 'url',
+  text: 'URL',
+}, {
+  key: 'body',
+  text: 'Body',
```

Comment:
key and text have the same value and it looks like text is only used in the html.  
maybe just have the one key `key` and then use `text-transform` to capitalize the html?

Reply:
I will do `text-transform: uppercase` otherwise  `Url` looks odd.